cmd/add-realm/main.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Adds a new realm.
+package main
+
+import (
+	"context"
+	"flag"
+	"log"
+
+	"github.com/google/exposure-notifications-verification-server/pkg/database"
+	"github.com/sethvargo/go-envconfig/pkg/envconfig"
+)
+
+func main() {
```

Comment:
It'd be good to use the `realMain()` pattern so we can properly cleanup things (defer's don't run with log.Fatal)

Reply:
Can be follow-up